extensions/install.php

```diff
@@ -1746,4 +1758,47 @@ protected function loadManifest($parent)
 
 		$this->manifest = $parent->get('manifest');
 	}
+
+	/**
+	 * Setup site url for redCORE config
+	 *
+	 * @return  void
+	 */
+	private function insertSiteDomain()
+	{
+		$db    = JFactory::getDbo();
```

Comment:
you should check first here if this is actually redCORE installation or not, because it will trigger on every extension install that has redCORE installer as parent class and that is almost all of them

Reply:
yes but it is needed to run it only once when redCORE updates because it is redCORE setting, otherwise it is just a waste of resources to run it with every module, plugin, ...

Reply:
I understand.

I modified code to run only when install redCORE component.